### PR TITLE
Updates ELM user guide

### PR DESCRIPTION
**Pull request by bishtgautam**
Adds a few notes in ELM's User Guide.

[BFB]

### CODE REVIEW

**Comment by bishtgautam** [comment]
The compsets mentioned in this documentation are defined in PR-https://github.com/E3SM-Project/E3SM/pull/6362.

File: components/elm/docs/user-guide/index.md [hunk 1]
```diff
@@ -1 +1,89 @@
-start of the ELM User's Guide
+# ELM User's Guide
+
+This User's Guide describes how to set up and run ELM.
+
+## Table of Conents
```
Comment (rljacob) [inline components/elm/docs/user-guide/index.md:5]: Remove this. A TOC is auto-generated on the right-hand-side of the page.

**Comment by rljacob** [comment]
Is this still WIP?  What else is planned?

**Comment by rljacob** [comment]
@bishtgautam we would like this merged before the tutorial starts on Tuesday.

File: components/elm/docs/user-guide/index.md (hunk 1 above)
Comment (bishtgautam) [reply components/elm/docs/user-guide/index.md]: Done.

**Comment by bishtgautam** [comment]
@acme-y9s Do you have any commits to push in this branch related to the BGC settings in ELM? If not, we can get this branch merged as is.

@rljacob, This PR needs
- https://github.com/E3SM-Project/E3SM/pull/6362 to be merged, and
- any new commit that @acme-y9s might have related to running the BGC configuration of ELM.

**Comment by thorntonpe** [comment]
Added a compset in the list of I cases for pre-industrial run with coupler output. Following @rljacob comment, removed the "v3" part from this compset name. @acme-y9s will need to modify the compset name in her PR to be 'I1850E3SMCNP'

**Review by rljacob — APPROVED** [review]

**Pull request merged**